App/pom.xml

```diff
@@ -29,6 +30,9 @@
                 <artifactId>javafx-maven-plugin</artifactId>
                 <version>${javafx-maven-plugin-version}</version>
                 <configuration>
+                    <options>
+                        <option>-Dcharm-desktop-form=${form.factor}</option>
```

Comment:
Do we need to pass a similar argument to the client plugin as well?

Reply:
This doesn't work yet for the client plugin. The runtime arguments have to be set directly in code (that's why I added it to the main class, commented out).